HISTORY.rst

```diff
@@ -5,6 +5,7 @@ History
 --------------------------------
 
 * Dev: New config for readthedocs
+* Feature: Add 'columns' parameter to 'get_dataframe' func (#1057)
```

Comment:
This can include a mention for `get_ticker_price` too.

Reply:
Done